src/import/import.rs

```diff
@@ -75,11 +75,11 @@ impl<Client: ImportClient> ImportJob<Client> {
 
         match res {
             Ok(_) => {
-                info!("{} takes {:?}", self.tag, start.elapsed());
+                info!("import engine succ"; "tag" => %self.tag, "takes" => ?start.elapsed());
```

Comment:
Is it better to remove the "succ" here? It is neither a common abbreviation nor necessary.